nl/mail/chrome/messenger/pbMessenger.dtd

```diff
@@ -367,43 +367,43 @@
 entities below were automatically added by l10nmerge
 -->
 
-<!ENTITY pbToolbarButton-runFiltersButton.label "Run Filters">
-<!ENTITY pbToolbarButton-runFiltersButton.tooltip "Run filters on the current folder">
+<!ENTITY pbToolbarButton-runFiltersButton.label "Filters Draaien">
+<!ENTITY pbToolbarButton-runFiltersButton.tooltip "Draai filters voor de huidige map">
```

Comment:
CORRECTION NEEDED, for this context it should be: ''Pas filters toe voor de huidige map''.